students/MikeShand/lesson03/mailroom.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+
+
+donors=[['Andy', [10.00]],['Bill',[15.00, 25.00]],['Chuck',[20.00,30.00,40.00]]]
+
+def donor_list():
+    donor_list=[]
+    for x in donors:
+        print(x[0])
+    thank_you()
+
+
+def thank_you():
+    print('Please enter the donor name\n (Type "list" for a list of current donor names)\n Press "q" to return to console')
+    cd=input(':')
+
+    if cd.lower()=='list':
+        donor_list()
+
+    elif cd.lower()=='q':
+        console()   
+
+    else:
+
+        amount=float(input('Please enter the donation amount:'))
+
+        for j,k in enumerate(donors):

```

Comment:
recommended to use more meaningful names: `for idx, donor in enumerate (donors)`

on another note how about a more simplified loop:
```
for name, donor_data in donors:
    if name == cd:
        donor_data.append(amount)
        break
else:
    donors.append([cd, [amount]])
```